[BUG]: Fatal Exception in collect activity if using percent trait that was imported without min or max

### Description

`2024-12-18 12:40:42.576  4502-4502  AndroidRuntime          com.fieldbook.tracker.debug          E  FATAL EXCEPTION: main
                                                                                                    Process: com.fieldbook.tracker.debug, PID: 4502
                                                                                                    java.lang.RuntimeException: Unable to resume activity {com.fieldbook.tracker.debug/com.fieldbook.tracker.activities.CollectActivity}: java.lang.NumberFormatException: For input string: ""
                                                                                                    	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4757)
                                                                                                    	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4790)
                                                                                                    	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:54)
                                                                                                    	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2210)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:201)
                                                                                                    	at android.os.Looper.loop(Looper.java:288)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:7839)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
                                                                                                    Caused by: java.lang.NumberFormatException: For input string: ""
                                                                                                    	at java.lang.Integer.parseInt(Integer.java:627)
                                                                                                    	at java.lang.Integer.parseInt(Integer.java:650)
                                                                                                    	at com.fieldbook.tracker.traits.PercentTraitLayout.afterLoadExists(PercentTraitLayout.java:147)
                                                                                                    	at com.fieldbook.tracker.traits.BaseTraitLayout.loadLayout(BaseTraitLayout.java:143)
                                                                                                    	at com.fieldbook.tracker.traits.PercentTraitLayout.loadLayout(PercentTraitLayout.java:97)
                                                                                                    	at com.fieldbook.tracker.views.TraitBoxView.loadLayout(TraitBoxView.kt:224)
                                                                                                    	at com.fieldbook.tracker.views.TraitBoxView.initTraitType(TraitBoxView.kt:168)
                                                                                                    	at com.fieldbook.tracker.activities.CollectActivity.initWidgets(CollectActivity.java:741)
                                                                                                    	at com.fieldbook.tracker.activities.CollectActivity.moveToResultCore(CollectActivity.java:855)
                                                                                                    	at com.fieldbook.tracker.activities.CollectActivity.moveToSearch(CollectActivity.java:829)
                                                                                                    	at com.fieldbook.tracker.activities.CollectActivity.onResume(CollectActivity.java:1005)
                                                                                                    	at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1476)
                                                                                                    	at android.app.Activity.performResume(Activity.java:8191)
                                                                                                    	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4747)`

### Expected behavior

_No response_

### Field Book version

6.0.2

--- a/app/src/main/java/com/fieldbook/tracker/traits/PercentTraitLayout.java
+++ b/app/src/main/java/com/fieldbook/tracker/traits/PercentTraitLayout.java
@@ -144,7 +144,10 @@ public class PercentTraitLayout extends BaseTraitLayout {
 
         if (value != null && !value.equals("NA") && !value.isEmpty()) {
 
-            seekBar.setMax(Integer.parseInt(getCurrentTrait().getMaximum()));
+            // Default to max 100 if maximum is not set
+            String maxString = getCurrentTrait().getMaximum();
+            int max = (maxString != null && !maxString.isEmpty()) ? Integer.parseInt(maxString) : 100;
+            seekBar.setMax(max);
 
             int textColor = value.equals(getDefaultValue()) ? Color.BLACK : Color.parseColor(getDisplayColor());
             setCurrentValueText(value, textColor);

